include/swift/Basic/Timer.h

```diff
@@ -33,7 +33,8 @@ namespace swift {
   public:
     explicit SharedTimer(StringRef name) {
       if (CompilationTimersEnabled == State::Enabled)
-        Timer.emplace(name, StringRef("Swift compilation"));
+        Timer.emplace(name, StringRef("Swift compilation"), StringRef("swift"),
```

Comment:
Yeah, I left them there for consistency.  The default constructor should work.  Id rather get this merged and then follow up with a cleanup patch to remove the explicit construction.